new-projects/readme.md

```diff
@@ -0,0 +1,19 @@
+# How to make a new project
+
+So you have an amazing idea that would benefit the client, or New Atoms to the moon and back? Great. Let's make a new project.
+
+### Get started
+To prevent our workflow from slowing down, there is no need to vote in achievables for making new projects.
+Instead, you can make a new project on your own accord.
+
+To do so, make a new folder in the projects folder of the relevant repository. Keep the folder name as short possible, yet descriptive. So rather than 'how-to-make-a-new-project', call it 'new-projects'. To this folder you add a) title.txt for the title of your project and b) readme.md, in which you put the description of you project.
+
```

Comment:
is readme in addition to pitch.md?

The interface will only display title.txt, pitch.md and post.md (and social posts).